director-configure-blobstore.html.md.erb

```diff
@@ -72,3 +72,23 @@ The Director and the Agents can use an S3 compatible blobstore. Here is how to c
         bucket_name: test-bosh-bucket
         host: objects.dreamhost.com
     ```
+
+---
+## <a id="gcs"></a> GCS
+
+The Director and the Agents can use GCS blobstore. Here is how to configure it:
+
+1. Create a GCS bucket.
+
+1. Ensure that access to the bucket is protected, as the Director may store sensitive information.
+
+1. Modify deployment manifest for the Director and specify GCS credentials and bucket name:
+
+    ```yaml
+    properties:
+      blobstore:
+        provider:
+        json_key: |
+          SERVICE-ACCOUNT-FILE
+        bucket_name: test-bosh-bucket
+    ```
```

Comment:
1. To use Customer Supplied Encryption Keys (link to CSEK page) to encrypt blobstore contents instead of server-side encryption keys specify `[property name]`

```yaml
example manifest with csek
```